### ISSUE TITLE
A11y_AzureDataStudio_Dashboard_High Contrast:In High Contrast Aquatic/ Desert theme, When keyboard focus lands on Home tab , Focus is not visible to users

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#22694]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #Win32; #WCAG4.3.1;  #High Contrast; #Benchmark; #No Disruption of Accessibility Features;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Environment and OS details:
Azure Data Studio Version: 1.44.0-insider (user setup)
Commit: 6a2ec12a3537725aaea52716c734114504760118
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Pre-Requisite for high contrast:

1. Open settings, tab till accessibility and press enter.
2. Tab till contrast theme list item and press enter.
3. Tab till contrast themes dropdown and press enter then select 'Aquatic/desert' theme.
4. Tab till 'apply' button and press enter.

### Repro Steps:

1. Launch Azure Data Studio.
2. Connect to server. tab till manage and press enter
3. Apply High Contrast
4. Verify after applying high contrast , when keyboard focus lands on Home tab , Focus should be visible to users or not
### Actual Result:
In High Contrast Aquatic/ Desert theme, when keyboard focus lands on Home tab , Focus is not visible to users
### Expected Result:
In High Contrast Aquatic/ Desert theme, when keyboard focus lands on Home tab , Focus should be visible to users
### Attachment:
![Media1](https://user-images.githubusercontent.com/100189407/231152526-84518b40-ea10-4f06-8c6d-3fb9bc805613.gif)

**Comment by alanrenmsft** [comment]
@VijayaBharathyA Please use the latest insider build to verify the fix.

**Comment by VijayaBharathyA** [comment]
#Closed; #Regressed:05-09-23;
Issue is fixed
Version: 1.44.0-insider (user setup)
Commit: ff3f6d53c7aecf551b308ed03333943194316009
Date: 2023-05-08T05:31:07.830Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
![Media1 (1)](https://user-images.githubusercontent.com/100189407/237026162-c4b9c5ce-7cf1-4d23-b20e-79729a5a9805.gif)
